src/interfaces/business-state-interfaces.ts

```diff
@@ -17,6 +17,16 @@ export interface AllowedActionsIF {
   }
 }
 
+/** The Alternate Name object. */
+export interface AlternateNameIF {
+  entityType: CorpTypeCd
+  identifier: string
+  name: string
+  nameRegisteredDate: ApiDateTimeUtc
+  nameStartDate: IsoDatePacific
+  nameType: NameTypes
+}
```

Comment:
When you do Edit UI, how about moving this to the shared interfaces (and coming back and importing here and in Filings UI)?